en_US/data-integration/data-bridge-mqtt.md

```diff
@@ -133,6 +147,16 @@ You have now successfully created the rule. You can see the newly created rule o
 
 You can also click **Integration** -> **Flow Designer** to view the topology. The topology visually represents how messages under the topic `t/#` are sent to the remote MQTT Broker after being processed by the rule `my_rule`.
 
+Note that, if using MQTTv5 protocol, publish properties are forwarded to the remote broker as is if the `pub_props` field is selected by the rule SQL, as in the `select * from t/#` example from this section.  If one wishes to dynamically add more user properties, one can just add then to the `pub_props` field that the rule outputs.  For example, the following rule will add an user property with key and value taken from the incoming payload:
+
+```sql
+SELECT
+  *,
+  map_put(concat('User-Property.', payload.extra_key), payload.extra_value, pub_props) as pub_props
+FROM
+  't/#'
+```
```

Comment:
This seems to have been duplicated above.  Maybe keep it in just one place?

Reply:
Sorry, I forgot to remove the original paragraph.